src/components/Card.js

```diff
@@ -23,27 +29,34 @@ export default class Card {
     );
 
     this._likeButton.addEventListener("click", () => {
-      this._handleLikeIcon();
+      this.handleLikeIcon();
```

Comment:
Needs correction: You should update the likes on the server. For that it's needed to declare the like handler in index.js provide it to the card constructor and use it as the like button listener.